funcionalities/cart.py

```diff
@@ -0,0 +1,107 @@
+from utils import get_database_connection
+import json
+from models.classes import Product, Buyer
+from fastapi import HTTPException
+
+with open(r"data\upiti.json") as f:
+    file = json.load(f)
+
+
+def view_cart():
+    connection = get_database_connection()
+    cursor = connection.cursor()
+    cursor.execute(file["select_cart"])
+    results = cursor.fetchall()
+    cursor.close()
+    connection.close()
```

Comment:
These 3 lines are used 4 times in exactly the same order 😄 Make some function and then use that function instead of calling the same 3 lines every time.